extensions/amp-bind/0.1/bind-impl.js

```diff
@@ -187,7 +190,9 @@ export class Bind {
     this.initializePromise_ =
         this.viewer_.whenFirstVisible().then(() => bodyPromise).then(body => {
           const head = (opt_win) ? opt_win.document.head : ampdoc.getHeadNode();
-          return this.initialize_(body, head && elementByTag(head, 'title'));
+          const title = head && elementByTag(head, 'title');
+          const fixedLayer = this.viewport_.getFixedLayerContainer();
+          return this.initialize_(body, fixedLayer, title);
```

Comment:
Why not use `document` as the root, instead of `body`? Then you don't need to be aware of the fixed layer.

Reply:
I wanted to avoid scanning the head unnecessarily, but this is a lot cleaner. Thanks.